app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -47,11 +47,11 @@ ListingController = ($scope, $state, $sce, $sanitize, $filter, Carousel, SharedS
   $scope.isFavorited = (listing_id) ->
     ListingService.isFavorited(listing_id)
 
-  $scope.formattedAddress = (listing) ->
-    ListingService.formattedAddress(listing)
+  $scope.formattedAddress = (listing, display) ->
```

Comment:
I feel like we should turn the address formatting into a filter one day. it works now but I think a filter is more applicable with the way we present the address